faqs/_posts/2016-03-30-what-are-xls-reports.md

```diff
@@ -0,0 +1,140 @@
+
+Here is the [XLS Report Template](https://docs.google.com/a/ona.io/spreadsheets/d/1bfeYJOWt72NBP_nZYruJOgEVIUvIgPTo2av4bjw67X4/edit#gid=965777567) created for the Mali Airports dataset.  You can use it to follow the instructions below.
+
+### Tags in an XLS Report Template
+
+In order to generate a report based on specific parameters, you must indicate on the template what information is to be extracted from the data set. You will use jXLS tags to pull in the raw data.
+
+<br>
+![](/content/screenshots/faq/xls-reports-1.png)
+
+A.  `</jx:forEach items= "${data}" var=”airport”>`is called the opening tag. In your XLS Report Template, replace "airport" with a name that is relevant to your dataset.
+
+After naming the dataset in the opening tag, you then order the dataset based on parameters collected in the form. 
+In the example above, we have ordered the “airport” dataset based on region, municipality, code, name, type, scheduled service and elevation parameters, which correspond to specific data fields in the form. 
+
+B.  `<jx:forEach>` is known as the closing tag. It invokes the end of the dataset.
+
+C.  `${airport.iso_region}` tag searches for region from ‘airport data’ and puts it into the corresponding cell.
+
+D.  `${airport.municipality}` tag searches for municipality from ‘airport data’ and puts it into the corresponding cell.
+
+E.  `${airport.ident}` tag searches for ident from ‘airport data’ and puts it into the corresponding cell.
+
+F.  `${airport.name}` tag searches for name from ‘airport data’ and puts it into the corresponding cell.
+
+G.  `${airport.type}` tag searches for type from ‘airport data’ and puts it into the corresponding cell.
+
+H.  `${airport.scheduled_service}` tag searches for scheduled service from ‘airport data’ and puts it into the corresponding cell.
+
+I.  `${airport.deviation_ft}` tag searches for deviation from ‘airport data’ and puts it into the corresponding cell.
+
+>
+**Note:** <br/>  If the parameter is within a group in the form, remember to include the group name in your tag, e.g. `${group_name_parameter_name}` 
+
+When you upload the template and generate a report, you will get the following [output](https://docs.google.com/a/ona.io/spreadsheets/d/16Lt-FgO-M0NwlNysQlVN_kw0y7__5242BocIL5ZBNsM/edit?usp=sharing):
+
+<br>
+![](/content/screenshots/faq/xls-reports-2.png)
+
+### How do I filter data in my report?
+
+You can introduce filters into an XLS Template so as to retrieve only specific information. If you would like to share specific subsets of data you can be able to retrieve that from your dataset. 
+
+For example, you have a dataset capturing number of mobile users in Africa; you can filter to view a dataset that only depicts mobile users in East Africa. You can as well filter the East African data set only to retrieve number of mobile users in Kenya.
```

Comment:
This example doesn't make a lot of sense, since users can't look at the dataset to see how the filter would work.